Code/Framework/AzFramework/Platform/Common/Xcb/AzFramework/XcbNativeWindow.cpp

```diff
@@ -219,17 +229,93 @@ namespace AzFramework
         }
     }
 
+    void XcbNativeWindow::DrawSplash()
+    {
+
+        int width = 0;
+        int height = 0;
+
+        auto get_image_data_static = [](int &width, int &height) -> uint8_t*
+        {
+            width = O3DE_LOGO::width;
+            height = O3DE_LOGO::height;
+
+            uint8_t *logo_binary_data = (uint8_t*)malloc(width * height * 4);   // so xcb_image_destroy can free it
+            
+            for (int i = 0; i < width * height; i++)
+            {
+                logo_binary_data[i * 4 + 0] = O3DE_LOGO::header_data_cmap[(unsigned char)O3DE_LOGO::header_data[i]][0];
+                logo_binary_data[i * 4 + 1] = O3DE_LOGO::header_data_cmap[(unsigned char)O3DE_LOGO::header_data[i]][1];
+                logo_binary_data[i * 4 + 2] = O3DE_LOGO::header_data_cmap[(unsigned char)O3DE_LOGO::header_data[i]][2];
+                logo_binary_data[i * 4 + 3] = 0xff;
+            }
+            return logo_binary_data;
+        };
+
+        uint8_t* data = get_image_data_static(width, height);
+
+        xcb_void_cookie_t cookie;
+        xcb_pixmap_t pixmap = xcb_generate_id(m_xcbConnection);
+        cookie = xcb_create_pixmap_checked(m_xcbConnection, m_xcbRootScreen->root_depth, pixmap, m_xcbWindow, width, height);
+        if (xcb_generic_error_t* error = xcb_request_check(m_xcbConnection, cookie))
+        {
+            AZ_Printf("XCB window init", "Error in xcb_create_pixmap: %d\n", error->error_code);
+            free(error);
+            free(data);
+            return;
+        }
+        // This uses hardcoded 8bit depth , TODO check on HDR monitor
+        xcb_image_t* image =
+            xcb_image_create_native(m_xcbConnection, width, height, XCB_IMAGE_FORMAT_Z_PIXMAP, 24, data, width * height * 4, data);
```

Comment:
As it was mentioned above:
```suggestion
            xcb_image_create_native(m_xcbConnection, width, height, XCB_IMAGE_FORMAT_Z_PIXMAP, 24, data, width * height * depth, data);
```
Please consider similar approach with 24 here.